libcontainer/factory_linux.go

```diff
@@ -160,6 +160,28 @@ func Load(root, id string) (*Container, error) {
 // to read the configuration and state. This is a low level implementation
 // detail of the reexec and should not be consumed externally.
 func StartInitialization() (err error) {
+	// Set up logging.
+	level := int(logrus.DebugLevel) // default to debug
+	// Passing log level is optional; currently libcontainer/integration does not do it.
+	if levelStr := os.Getenv("_LIBCONTAINER_LOGLEVEL"); levelStr != "" {
+		level, err = strconv.Atoi(levelStr)
+		if err != nil {
+			panic(fmt.Errorf("unable to convert _LIBCONTAINER_LOGLEVEL: %w", err))
```

Comment:
Can we change these to return error instead of panicking? Previously this code was in `init()` which required a panic, but here we can just return if I guess

Reply:
Done